packedForest/src/fpSingleton/fpInfo.h

```diff
@@ -71,6 +82,13 @@ namespace fp {
 				float_epsilon=std::numeric_limits<float>::epsilon()*10;
 				seed=-1;
 				numTreeBins=-1;
+				methodToUse = 1; // Should this default to 1?
```

Comment:
add these parameter defaults to the constructor

line ~258

Reply:
I now get these warnings:

```
Synapse:packedForest JLP$ make
g++-8 -std=c++11 -fopenmp -Wall -O3 -DNDEBUG -ffast-math -Iinc -I/usr/local/include  -c -o obj/fp.o src/fp.cpp
In file included from src/fpSingleton/fpData.h:7,
                 from src/fpSingleton/fpSingleton.h:4,
                 from src/packedForest.h:22,
                 from src/fp.cpp:1:
src/fpSingleton/fpInfo.h: In constructor 'fp::fpInfo::fpInfo()':
src/fpSingleton/fpInfo.h:61:9: warning: 'fp::fpInfo::useRowMajor' will be initialized after [-Wreorder]
    bool useRowMajor;
         ^~~~~~~~~~~
src/fpSingleton/fpInfo.h:31:8: warning:   'int fp::fpInfo::methodToUse' [-Wreorder]
    int methodToUse;
        ^~~~~~~~~~~
src/fpSingleton/fpInfo.h:255:4: warning:   when initialized here [-Wreorder]
    fpInfo(): numTreesInForest(100),
    ^~~~~~
src/fpSingleton/fpInfo.h:37:8: warning: 'fp::fpInfo::patchHeightMax' will be initialized after [-Wreorder]
    int patchHeightMax;
        ^~~~~~~~~~~~~~
src/fpSingleton/fpInfo.h:36:8: warning:   'int fp::fpInfo::patchHeightMin' [-Wreorder]
    int patchHeightMin;
        ^~~~~~~~~~~~~~
src/fpSingleton/fpInfo.h:255:4: warning:   when initialized here [-Wreorder]
    fpInfo(): numTreesInForest(100),
    ^~~~~~
src/fpSingleton/fpInfo.h:39:8: warning: 'fp::fpInfo::patchWidthMax' will be initialized after [-Wreorder]
    int patchWidthMax;
        ^~~~~~~~~~~~~
src/fpSingleton/fpInfo.h:38:8: warning:   'int fp::fpInfo::patchWidthMin' [-Wreorder]
    int patchWidthMin;
        ^~~~~~~~~~~~~
src/fpSingleton/fpInfo.h:255:4: warning:   when initialized here [-Wreorder]
    fpInfo(): numTreesInForest(100),
    ^~~~~~
g++-8 -std=c++11 -o bin/fp obj/fp.o -fopenmp -lm -O3 
```

Reply:
I fixed it.  I ordered them incorrectly